libr/bin/demangle.c

```diff
@@ -62,7 +62,7 @@ R_API int r_bin_demangle_type(const char *str) {
 	return R_BIN_NM_NONE;
 }
 
-R_API char *r_bin_demangle(RBinFile *bf, const char *def, const char *str, ut64 vaddr) {
+R_API char *r_bin_demangle(RBinFile *bf, const char *def, const char *str, ut64 vaddr, bool libs) {
 	int type = -1;
```

Comment:
this function proabbly requires an `r_return_val_if_fail (bf && def && str, NULL);`

Reply:
some places call this function with def = NULL, so i've used only `bf && str`

Reply:
having def = NULL lgtm, probably only chking for bf and str should be good, but you did removed it completely

Reply:
bf can be null as well if bin info wasnt loaded, so do i just check str?